src/content/self-hosted/manage/airgap.mdx

```diff
@@ -0,0 +1,117 @@
+
+### Push Images to your Private Registry
+
+For simplicity, we recommend using the same repository names and tags in your private registry.
+
+:::info
+Okteto doesn't support `imagePullSecrets`, so your cluster must be configured to have access to your Private Registry, 
+:::
+
+**Required Images for Okteto Platform Installation**
+
+Push the following images to your private registry for the Okteto Chart installation:
+
+- **okteto/backend:{variables.chartVersion}**
+- **okteto/frontend:{variables.chartVersion}**
+- **okteto/buildkit:{variables.chartVersion}**
+- **okteto/registry:{variables.chartVersion}**
+- **okteto/pipeline-installer:{variables.chartVersion}**
+- **okteto/pipeline-runner:{variables.chartVersion}**
+- **okteto/ingress-nginx:{variables.chartVersion}**
+- **okteto/reloader:{variables.chartVersion}**
+- **okteto/redis:{variables.chartVersion}**
+
+For each Okteto CLI version, push the following images to your private registry:
+
+- **okteto/okteto:{variables.cliVersion}**
+- **okteto/dockerfile:{variables.cliVersion}**
+
+:::info
+For Okteto Chart {variables.chartVersion}, we recommend enforcing the usage of Okteto CLI {variables.cliVersion} in your developer's machines to reduce maintenance overhead
+:::
+
+## Step 3: Update Helm Configuration
+
+Once your images are pushed to your private registry, update your Okteto Helm configuration as follows:
+
+<CodeBlock language="yaml">
+  {`globals:
+  registry: <<your-registry-url>>
+
+ingress-nginx:
+  image:
+    image: <<your-registry-url>>/okteto/ingress-nginx
+    tag: ${variables.chartVersion}
+
+okteto-nginx:
+  image:
+    image: <<your-registry-url>>/okteto/ingress-nginx
+    tag: ${variables.chartVersion}
+
+redis:
+  image:
+    registry: <<your-registry-url>>
+    repository: okteto/reloader
+    tag: ${variables.chartVersion}
+
+reloader:
+  deployment:
+    image:
+      name: <<your-registry-url>>/okteto/reloader
+      tag: ${variables.chartVersion}
+`}
+</CodeBlock>
+
+
+## Step 4: Install or Upgrade Okteto 
+
+Execute the following command to install or upgrade your Okteto instance:
+
+<CodeBlock language="bash">
+  {`helm upgrade --install okteto okteto/okteto -f config.yaml --namespace=okteto --create-namespace --version=${variables.chartVersion}`}
+</CodeBlock>
+
+## Optional: Use Your Own Ingress Controller
+
+By default, Okteto deploy an [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/) to expose the Okteto Control Plane and the applications managed by Okteto.
+Our NGINX Ingress Controller is usually exposed in a dedicated Load Balancer.
+
+Refer to [this community guide](https://community.okteto.com/t/how-do-i-configure-my-own-ingress-controller/887) for detailed instructions on disabling the installation of our Nginx Ingress Controller and exposing Okteto behind your own Ingress Controller.
+
+## Install Okteto without a Wildcard Certificate (optional)
```

Comment:
Is this truly optional on Air Gapped environments? I feel that this recommendation should be part of the Certificates page, since this might be the case even in other types of installations.

Reply:
It's more common on air-gapped. We can add it to the certificates section, but I call out here will also help